Parallel blockchain tests (#375)
bitcoindevkit/bdk: README.md

}
```

## Testing

### Unit testing

```
cargo test
```

### Integration testing

Integration testing require testing features, for example:

```

> cargo test --features test-electrum

Reviewer: 💡  Can the feature flag situation be improved? How about `test-blockchains` + `electrum` does this and we can get rid of `test-electrum`.

Author: The problem to do this is the different testing features required by esplora (`electrsd/esplora_a33e97e1`) and electrum (`electrsd/electrs_0_8_10`)

Previously, I even though to remove `test-*` completely (and simply run blockchain electrum test with electrum feature), in this case other than previously described issue there was another issue to include dep like `bitcoincore-rpc` only when testing.

```

The other options are `test-esplora` or `test-rpc`.

> Note that `electrs` and `bitcoind` binaries are automatically downloaded (on mac and linux), to specify your own one must use `--no-default-features` and provide `BITCOIND_EXE` and `ELECTRS_EXE` as environment variables.

Reviewer 2: nit: 
```suggestion
Note that `electrs` and `bitcoind` binaries are automatically downloaded (on mac and linux), to specify you already have installed binaries you must use `--no-default-features` and provide `BITCOIND_EXE` and `ELECTRS_EXE` as environment variables.
```
